Yank/repex.py

```diff
@@ -1575,6 +1659,59 @@ def from_storage(cls, storage):
         # Don't write the new last iteration, we have not technically written anything yet, so there is no "junk"
         return repex
 
+    # TODO use Python 3.6 namedtuple syntax when we drop Python 3.5 support.
+    Status = typing.NamedTuple('Status', [
+        ('iteration', int),
+        ('target_error', float),
+        ('is_completed', bool)
+    ])
+
+    @classmethod
+    def read_status(cls, storage):
```

Comment:
Question, what happens if you open a netcdf file for read while its also open for write from a running simulation, and then what happens when the read-opened file is closed?

Reply:
I think it's fine. I tried it on my running simulations on the cluster and it worked correctly.